test/fixtures/special_chars.ftl

```diff
@@ -0,0 +1,12 @@
+bracket-inline = [Value]
```

Comment:
You're missing the 

```properties
## OK

```

Reply:
Grr, sorry, my bad. I removed it to re-trigger the build and forgot to re-add it :/